experiments/AtmosLES/dycoms.jl

```diff
@@ -360,6 +361,12 @@ function config_dycoms(FT, N, resolution, xmax, ymax, zmax)
     return config
 end
 
+function config_diagnostics(driver_config)
+    interval = 10000 # in time steps
+    dgngrp = setup_atmos_default_diagnostics(interval, driver_config.name)
```

Comment:
Great that the output file name contains basic information on the test. 
It would be useful if it also contained info on the number of elements in each direction and even the total number of time steps contained in the file.

Reply:
Also also the SGS model name. E.g. "SGS_SMAGO" or "SGS_Vreman" etc.

Reply:
This being said, a simple ascii log file with the basic information of the simulation setup would be extremely useful for repeatability.

Reply:
I think this generally refers to making the properties of `AtmosModel` available in some way ?

Reply:
The Parameters module is the first step towards achieving exactly the goal of repeatability. There is some ways to go though...

Reply:
Are we able to tie in the `config_diagnostics` interval to the `simtime` instead of timestep count ? The timestep count is useful for de-bugging, but for diagnostics retrieval the simulation time is perhaps more intuitive ?

Reply:
As we're using the generic callbacks mechanism, we can use any of `EveryXWallTimeSeconds`, `EveryXSimulationTime`, and `EveryXSimulationSteps`. Currently the driver uses `EveryXSimulationSteps`. We can either change that, or add the ability to choose any of the mechanisms.

Reply:
Yeah I see https://github.com/climate-machine/CLIMA/blob/fc4f374dc5612298c93fc3cf95f257b6311045f0/src/ODESolvers/GenericCallbacks.jl#L54 the commented out section is the `simtime` callback hook we would need